gdnative-derive/src/lib.rs

```diff
@@ -16,6 +16,39 @@ mod profiled;
 mod varargs;
 mod variant;
 
+/// Collects method signatures of all functions in a `NativeClass` that have the `#[export]` attribute and registers them with godot.
```

Comment:
Maybe add something like:

> For more context, consult the documentation of the [`NativeClass`](link) derive macro.

We recently upgraded rustc to 1.54, so we can use [doc links](https://doc.rust-lang.org/rustdoc/linking-to-items-by-name.html. There are two `NativeClass` symbol, the class and the proc macro, I'd use the latter here.

Also, please capitalize "Godot" 🙂